snow/engine/snowman/transitive_test.go

```diff
@@ -30,10 +31,36 @@ var (
 	errInvalid        = errors.New("invalid")
 	errUnexpectedCall = errors.New("unexpected call")
 	errTest           = errors.New("non-nil test")
-	Genesis           = ids.GenerateTestID()
+
+	GenesisID    = ids.GenerateTestID()
+	GenesisBytes = utils.RandomBytes(32)
+	Genesis      = &snowman.TestBlock{
+		TestDecidable: choices.TestDecidable{
+			IDV:     GenesisID,
+			StatusV: choices.Accepted,
+		},
+		BytesV: GenesisBytes,
+	}
 )
 
-func setup(t *testing.T, engCfg Config) (ids.NodeID, validators.Manager, *common.SenderTest, *block.TestVM, *Transitive, snowman.Block) {
+func BuildChain(root *snowman.TestBlock, length int) []*snowman.TestBlock {
+	chain := make([]*snowman.TestBlock, length)
+	for i := range chain {
+		root = &snowman.TestBlock{
+			TestDecidable: choices.TestDecidable{
+				IDV:     ids.GenerateTestID(),
+				StatusV: choices.Processing,
+			},
+			ParentV: root.ID(),
+			HeightV: root.Height() + 1,
+			BytesV:  utils.RandomBytes(32),
+		}
+		chain[i] = root
+	}
+	return chain
+}
+
+func setup(t *testing.T, engCfg Config) (ids.NodeID, validators.Manager, *common.SenderTest, *block.TestVM, *Transitive) {
```

Comment:
seems that `setup` is only called by `setupDefaultConfig`. Consider merging the two functions into a single `setup`